www/src/scss/stub-theme/_overrides.scss

```diff
@@ -0,0 +1,2 @@
+// Use this file to define style overrides for @edx/paragon
```

Comment:
Do we want to include this here? Or encourage people to make a brand repo? Isn't adding code here sorta the 'wrong' way to do it, since they're effectively editing the paragon repository instead of creating a brand to alias?

Reply:
Oh, I guess it's an example/stub theme... question still sorta stands, but in the sense that maybe it should just direct folks to instructions on how to pull this example/stub out and use it 'properly'...

Reply:
Personally, I like having the example/stub themes; it shows me exactly where things need to be in the file structure in a way that is often less intuitive when I'm just reading the docs.

Reply:
The comment may be wrong. The reason this is here is because in order to swap themes I need to include something even if in the default case I don't want it. Maybe `dummy-theme` is a better name.